### PR TITLE
CAMEL-19573 - Stop syncing version properties in git, generate the effective camel-dependencies during the build

### PR DESCRIPTION
**Pull request by ppalaga**
https://issues.apache.org/jira/browse/CAMEL-19573

### CODE REVIEW

**Comment by oscerd** [comment]
Also If I remember correctly, there were other attempts to do this but the release was failing, so maybe we need to test also with a DryRun of release:prepare

**Comment by ppalaga** [comment]
I am quite sure installing works as expected. There should be no big difference when deploying. Let my try it.

**Comment by oscerd** [comment]
It's just my mind remembering this, but I could be wrong.

File: tooling/maven/sync-properties-maven-plugin/src/main/resources/camel-dependencies-template.xml [hunk 1]
```diff
@@ -34,32 +34,23 @@
...
-                </pluginManagement>
-            </build>
-        </profile>
-    </profiles>
+    <url>https://camel.apache.org</url>
+    <inceptionYear>2007</inceptionYear>
+    <scm>
+        <connection>scm:git:http://gitbox.apache.org/repos/asf/camel.git</connection>
+        <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/camel.git</developerConnection>
+        <url>https://gitbox.apache.org/repos/asf?p=camel.git;a=summary</url>
```
Comment (essobedo) [inline tooling/maven/sync-properties-maven-plugin/src/main/resources/camel-dependencies-template.xml:48]: Are they still actual?

Comment (oscerd) [reply tooling/maven/sync-properties-maven-plugin/src/main/resources/camel-dependencies-template.xml:48]: gitbox redirects on github now, I think they are trying to find a fix in infra, but it's not done yet.

**Comment by ppalaga** [comment]
I have cut a fake relase 4.0.0-test with a real staging repo where camel-dependencies was deployed as expected: https://repository.apache.org/content/repositories/orgapachecamel-1588/org/apache/camel/camel-dependencies/4.0.0-test/camel-dependencies-4.0.0-test.pom (I'll drop the staging repo without releasing soon) Here is the content:

```
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">

    <modelVersion>4.0.0</modelVersion>
    <parent>
        <groupId>org.apache</groupId>
        <artifactId>apache</artifactId>
        <version>25</version>
        <relativePath />
    </parent>

    <groupId>org.apache.camel</groupId>
    <artifactId>camel-dependencies</artifactId>
    <version>4.0.0-test</version>

    <packaging>pom</packaging>
    <name>Camel :: Dependencies</name>
    <description>Version properties used for building and testing Apache Camel</description>

    <properties>
        <abdera-version>1.1.3</abdera-version>
        <activemq-artemis-version>2.28.0</activemq-artemis-version>
        <activemq-version>5.18.1</activemq-version>
        <angus-mail-version>2.0.2</angus-mail-version>
        <apache-drill-version>1.21.1</apache-drill-version>
        <apacheds-version>2.0.0.AM26</apacheds-version>
        <apicurio-version>1.1.27</apicurio-version>
        <arangodb-java-version>6.20.0</arangodb-java-version>
        <as2-lib-version>4.11.0</as2-lib-version>
        <ascii-table-version>1.8.0</ascii-table-version>
        <asm-version>8.0.1</asm-version>
        <asn1bean-version>1.14.0</asn1bean-version>
        <assertj-version>3.24.2</assertj-version>
        <asterisk-java-version>3.37.0</asterisk-java-version>
        <atlasmap-version>2.5.2</atlasmap-version>
        <atlassian-fugue-version>3.0.0</atlassian-fugue-version>
        <atmosphere-version>3.0.3</atmosphere-version>
        <avro-ipc-jetty-version>1.11.1</avro-ipc-jetty-version>
        <avro-ipc-netty-version>1.11.1</avro-ipc-netty-version>
        <avro-version>1.11.1</avro-version>
        <awaitility-version>4.2.0</awaitility-version>
        <aws-java-sdk2-version>2.20.98</aws-java-sdk2-version>
        <aws-xray-version>2.14.0</aws-xray-version>
        <axiom-version>1.2.14</axiom-version>
        <azure-sdk-bom-version>1.2.14</azure-sdk-bom-version>
        <azure-storage-blob-changefeed-version>12.0.0-beta.18</azure-storage-blob-changefeed-version>
        <bouncycastle-version>1.74</bouncycastle-version>
        <box-java-sdk-version>2.58.0</box-java-sdk-version>
        <braintree-gateway-version>3.24.0</braintree-gateway-version>
        <build-helper-maven-plugin-version>3.4.0</build-helper-maven-plugin-version>
        <c3p0-version>0.9.5.5</c3p0-version>
        <caffeine-version>3.1.6</caffeine-version>
        <californium-scandium-version>2.7.4</californium-scandium-version>
        <californium-version>2.7.2</californium-version>
        <cassandra-driver-version>4.16.0</cassandra-driver-version>
        <cglib-version>3.2.12</cglib-version>
        <chunk-templates-version>3.6.2</chunk-templates-version>
        <classgraph-version>4.8.157</classgraph-version>
        <cobertura-maven-plugin-version>2.7</cobertura-maven-plugin-version>
        <cometd-java-client-version>7.0.9</cometd-java-client-version>
        <cometd-java-server-version>7.0.9</cometd-java-server-version>
        <commons-beanutils-version>1.9.4</commons-beanutils-version>
        <commons-cli-version>1.4</commons-cli-version>
        <commons-codec-version>1.16.0</commons-codec-version>
        <commons-collections-version>3.2.2</commons-collections-version>
        <commons-collections4-version>4.4</commons-collections4-version>
        <commons-compress-version>1.21</commons-compress-version>
        <commons-configuration-version>1.9</commons-configuration-version>
        <commons-configuration2-version>2.8.0</commons-configuration2-version>
        <commons-csv-version>1.8</commons-csv-version>
        <commons-dbcp-version>1.4</commons-dbcp-version>
        <commons-dbcp2-version>2.9.0</commons-dbcp2-version>
        <commons-exec-version>1.3</commons-exec-version>
        <commons-io-version>2.13.0</commons-io-version>
        <commons-lang-version>2.6</commons-lang-version>
        <commons-lang3-version>3.12.0</commons-lang3-version>
        <commons-logging-version>1.2</commons-logging-version>
        <commons-math3-version>3.6.1</commons-math3-version>
        <commons-net-version>3.9.0</commons-net-version>
        <commons-pool2-version>2.11.1</commons-pool2-version>
        <commons-text-version>1.10.0</commons-text-version>
        <commons-validator-version>1.7</commons-validator-version>
        <compress-lzf-version>1.1</compress-lzf-version>
        <consul-client-version>1.5.3</consul-client-version>
        <corda-version>4.10</corda-version>
        <couchbase-client-version>3.4.7</couchbase-client-version>
        <curator-version>5.4.0</curator-version>
        <cxf-codegen-plugin-version>4.0.2</cxf-codegen-plugin-version>
        <cxf-version>4.0.2</cxf-version>
        <cxf-xjc-plugin-version>4.0.0</cxf-xjc-plugin-version>
        <cxf-xjc-utils-version>4.0.0</cxf-xjc-utils-version>
        <datasonnet-mapper-version>2.5.2-jakarta4</datasonnet-mapper-version>
        <debezium-mysql-connector-version>8.0.33</debezium-mysql-connector-version>
        <debezium-version>2.3.0.Final</debezium-version>
        <depends-maven-plugin-version>1.4.0</depends-maven-plugin-version>
        <derby-version>10.14.2.0</derby-version>
        <digitalocean-api-client-version>2.17</digitalocean-api-client-version>
        <directory-watcher-version>0.18.0</directory-watcher-version>
        <disruptor-version>3.4.4</disruptor-version>
        <djl-mxnet-native-version>1.8.0</djl-mxnet-native-version>
        <djl-pytorch-native-version>1.9.1</djl-pytorch-native-version>
        <djl-tensorflow-native-version>2.4.1</djl-tensorflow-native-version>
        <djl-version>0.21.0</djl-version>
        <dnsjava-version>3.5.2</dnsjava-version>
        <docker-java-version>3.3.1</docker-java-version>
        <dropbox-version>5.4.5</dropbox-version>
        <eddsa-version>0.3.0</eddsa-version>
        <egit-github-core-version>2.1.5</egit-github-core-version>
        <ehcache3-version>3.10.8</ehcache3-version>
        <elasticsearch-java-client-sniffer-version>8.8.1</elasticsearch-java-client-sniffer-version>
        <elasticsearch-java-client-version>8.8.1</elasticsearch-java-client-version>
        <exec-maven-plugin-version>3.1.0</exec-maven-plugin-version>
        <facebook4j-core-version>2.4.13</facebook4j-core-version>
        <fastjson-version>2.0.34</fastjson-version>
        <findbugs-maven-plugin-version>3.0.5</findbugs-maven-plugin-version>
        <flatpack-version>4.0.5</flatpack-version>
        <flink-version>1.17.1</flink-version>
        <fop-version>2.8</fop-version>
        <formatter-maven-plugin-version>2.21.0</formatter-maven-plugin-version>
        <freemarker-version>2.3.32</freemarker-version>
        <ftpserver-version>1.1.2</ftpserver-version>
        <geronimo-annotation-1.0-spec-version>1.1.1</geronimo-annotation-1.0-spec-version>
        <geronimo-annotation-1.2-spec-version>1.0</geronimo-annotation-1.2-spec-version>
        <geronimo-atinject-1.0-spec-version>1.0</geronimo-atinject-1.0-spec-version>
        <geronimo-el-spec-version>1.0.1</geronimo-el-spec-version>
        <geronimo-interceptor-1.1-spec-version>1.0</geronimo-interceptor-1.1-spec-version>
        <geronimo-interceptor-1.2-spec-version>1.0</geronimo-interceptor-1.2-spec-version>
        <geronimo-j2ee-connector-spec-version>2.0.0</geronimo-j2ee-connector-spec-version>
        <geronimo-j2ee-jacc-spec-version>1.1</geronimo-j2ee-jacc-spec-version>
        <geronimo-j2ee-management-spec-version>1.1</geronimo-j2ee-management-spec-version>
        <geronimo-jcdi-1.0-spec-version>1.0</geronimo-jcdi-1.0-spec-version>
        <geronimo-jcdi-1.1-spec-version>1.0</geronimo-jcdi-1.1-spec-version>
        <glassfish-jaxb-runtime-version>4.0.1</glassfish-jaxb-runtime-version>
        <gmavenplus-plugin-version>2.1.0</gmavenplus-plugin-version>
        <google-api-client-version>1.35.2</google-api-client-version>
        <google-api-services-calendar-version>v3-rev20211229-1.32.1</google-api-services-calendar-version>
        <google-api-services-drive-version>v2-rev20220110-1.32.1</google-api-services-drive-version>
        <google-api-services-mail-version>v1-rev20211108-1.32.1</google-api-services-mail-version>
        <google-api-services-sheets-version>v4-rev20210629-1.32.1</google-api-services-sheets-version>
        <google-auth-library-oauth2-http-version>1.7.0</google-auth-library-oauth2-http-version>
        <google-cloud-bom-version>26.1.6</google-cloud-bom-version>
        <google-cloud-functions-bom-version>2.2.0</google-cloud-functions-bom-version>
        <google-cloud-functions-gax-grpc-version>1.62.0</google-cloud-functions-gax-grpc-version>
        <google-cloud-secretmanager-bom-version>2.2.0</google-cloud-secretmanager-bom-version>
        <google-maps-services-version>2.2.0</google-maps-services-version>
        <google-oauth-client-version>1.34.0</google-oauth-client-version>
        <graaljs-version>22.3.2</graaljs-version>
        <graphql-java-version>18.4</graphql-java-version>
        <greenmail-version>2.0.0</greenmail-version>
        <grizzly-websockets-version>2.4.4</grizzly-websockets-version>
        <grpc-google-auth-library-version>1.5.3</grpc-google-auth-library-version>
        <grpc-java-jwt-version>4.3.0</grpc-java-jwt-version>
        <grpc-netty-tcnative-boringssl-static-version>2.0.59.Final</grpc-netty-tcnative-boringssl-static-version>
        <grpc-version>1.56.0</grpc-version>
        <gson-version>2.10.1</gson-version>
        <guava-version>32.0.1-jre</guava-version>
        <guice3-version>3.0</guice3-version>
        <hadoop3-version>3.3.6</hadoop3-version>
        <hamcrest-version>2.2</hamcrest-version>
        <hapi-base-version>2.3</hapi-base-version>
        <hapi-fhir-version>6.4.2</hapi-fhir-version>
        <hapi-version>2.3</hapi-version>
        <hazelcast-version>5.3.1</hazelcast-version>
        <hdrhistrogram-version>2.1.11</hdrhistrogram-version>
        <hibernate-validator-version>8.0.0.Final</hibernate-validator-version>
        <hibernate-version>5.6.15.Final</hibernate-version>
        <hk2-version>2.6.1</hk2-version>
        <hsqldb-version>2.7.1</hsqldb-version>
        <httpasyncclient-version>4.1.5</httpasyncclient-version>
        <httpclient-version>5.1.4</httpclient-version>
        <httpclient4-version>4.5.14</httpclient4-version>
        <httpcore-version>5.2.2</httpcore-version>
        <httpcore4-version>4.4.16</httpcore4-version>
        <httpunit-version>1.7</httpunit-version>
        <huaweicloud-obs-version>3.23.3.1</huaweicloud-obs-version>
        <huaweicloud-sdk-version>3.1.45</huaweicloud-sdk-version>
        <ical4j-version>1.0.7</ical4j-version>
        <icu4j-version>73.1</icu4j-version>
        <ignite-version>2.15.0</ignite-version>
        <impsort-maven-plugin-version>1.9.0</impsort-maven-plugin-version>
        <infinispan-version>14.0.11.Final</infinispan-version>
        <influx-client-java-driver-version>6.8.0</influx-client-java-driver-version>
        <influx-java-driver-version>2.22</influx-java-driver-version>
        <irclib-version>1.10</irclib-version>
        <ironmq-version>3.0.5</ironmq-version>
        <ivy-version>2.5.1</ivy-version>
        <jackrabbit-version>2.21.16</jackrabbit-version>
        <jackson-jq-version>1.0.0-preview.20230409</jackson-jq-version>
        <jackson2-module-scala-version>2.15.1</jackson2-module-scala-version>
        <jackson2-version>2.15.2</jackson2-version>
        <jackson2.14-version>2.14.3</jackson2.14-version>
        <jakarta-activation-version>2.1.1</jakarta-activation-version>
        <jakarta-annotation-api-version>2.1.1</jakarta-annotation-api-version>
        <jakarta-api-version>2.1.5</jakarta-api-version>
        <jakarta-el-api-hibernate-validator-version>5.0.1</jakarta-el-api-hibernate-validator-version>
        <jakarta-el-expressly-version>5.0.0</jakarta-el-expressly-version>
        <jakarta-enterprise-cdi-api-version>4.0.1</jakarta-enterprise-cdi-api-version>
        <jakarta-inject-version>2.0.1</jakarta-inject-version>
        <jakarta-jms-api-version>3.1.0</jakarta-jms-api-version>
        <jakarta-json-api-version>2.1.1</jakarta-json-api-version>
        <jakarta-json-bind-api-version>3.0.0</jakarta-json-bind-api-version>
        <jakarta-jws-api-version>3.0.0</jakarta-jws-api-version>
        <jakarta-persistence-api-version>3.1.0</jakarta-persistence-api-version>
        <jakarta-servlet-api-version>6.0.0</jakarta-servlet-api-version>
        <jakarta-transaction-api-version>2.0.1</jakarta-transaction-api-version>
        <jakarta-xml-bind-api-version>4.0.0</jakarta-xml-bind-api-version>
        <jakarta-xml-soap-api-version>3.0.0</jakarta-xml-soap-api-version>
        <jakarta-xml-ws-api-version>4.0.0</jakarta-xml-ws-api-version>
        <jakarta.el-version>3.0.3</jakarta.el-version>
        <jandex-version>3.1.1</jandex-version>
        <jansi-version>2.4.0</jansi-version>
        <jasminb-jsonapi-version>0.12</jasminb-jsonapi-version>
        <jasypt-version>1.9.3</jasypt-version>
        <java-grok-version>0.1.9</java-grok-version>
        <java-util-version>2.0.0</java-util-version>
        <javacc-maven-plugin-version>3.0.1</javacc-maven-plugin-version>
        <javacrumbs-version>0.22</javacrumbs-version>
        <javaparser-version>3.13.10</javaparser-version>
        <javapoet-version>1.11.1</javapoet-version>
        <javassist-version>3.28.0-GA</javassist-version>
        <javax-el3-version>3.0.0</javax-el3-version>
        <jaxb2-maven-plugin-version>3.1.0</jaxb2-maven-plugin-version>
        <jaxp-ri-version>1.4.5</jaxp-ri-version>
        <jboss-el-api_3.0_spec-version>2.0.0.Final</jboss-el-api_3.0_spec-version>
        <jboss-logging-version>3.4.3.Final</jboss-logging-version>
        <jboss-marshalling-version>1.4.10.Final</jboss-marshalling-version>
        <jboss-xnio-version>3.3.8.Final</jboss-xnio-version>
        <jcache-version>1.1.1</jcache-version>
        <jcommander-version>1.72</jcommander-version>
        <jcr-version>2.0</jcr-version>
        <jedis-client-version>4.4.2</jedis-client-version>
        <jetcd-version>0.7.5</jetcd-version>
        <jettison-version>1.5.4</jettison-version>
        <jetty-plugin-version>${jetty-version}</jetty-plugin-version>
        <jetty-version>11.0.15</jetty-version>
        <jgit-version>6.5.0.202303070854-r</jgit-version>
        <jgroups-raft-leveldbjni-version>1.8</jgroups-raft-leveldbjni-version>
        <jgroups-raft-mapdb-version>1.0.8</jgroups-raft-mapdb-version>
        <jgroups-raft-version>0.5.3.Final</jgroups-raft-version>
        <jgroups-version>4.2.17.Final</jgroups-version>
        <jira-rest-client-api-version>5.2.5</jira-rest-client-api-version>
        <jnats-version>2.16.12</jnats-version>
        <jodatime2-version>2.11.1</jodatime2-version>
        <johnzon-version>1.2.20</johnzon-version>
        <jolokia-version>1.7.2</jolokia-version>
        <jolt-version>0.1.8</jolt-version>
        <jool-version>0.9.15</jool-version>
        <jooq-version>3.18.4</jooq-version>
        <joor-version>0.9.14</joor-version>
        <jsch-version>0.2.9</jsch-version>
        <jslt-version>0.1.14</jslt-version>
        <jsmpp-version>2.3.11</jsmpp-version>
        <json-patch-version>1.13</json-patch-version>
        <json-path-version>2.8.0</json-path-version>
        <json-schema-validator-version>2.2.14</json-schema-validator-version>
        <json-unit-version>2.32.0</json-unit-version>
        <jsonassert-version>1.5.1</jsonassert-version>
        <jsonata4java-version>2.2.4</jsonata4java-version>
        <jsoup-version>1.15.3</jsoup-version>
        <jt400-version>20.0.0</jt400-version>
        <jta-api-1.2-version>1.2</jta-api-1.2-version>
        <junit-jupiter-version>5.9.2</junit-jupiter-version>
        <junit-pioneer-version>1.7.1</junit-pioneer-version>
        <junit-toolbox-version>2.4</junit-toolbox-version>
        <junit-version>4.13.2</junit-version>
        <jxmpp-version>0.6.4</jxmpp-version>
        <jython-standalone-version>2.7.3</jython-standalone-version>
        <jython-version>2.7.3</jython-version>
        <jzlib-version>1.1.3</jzlib-version>
        <kafka-version>3.5.0</kafka-version>
        <kotlin-version>1.8.22</kotlin-version>
        <kubernetes-client-version>6.7.2</kubernetes-client-version>
        <kubernetes-model-version>6.7.2</kubernetes-model-version>
        <kudu-version>1.16.0</kudu-version>
        <leveldb-api-version>0.12</leveldb-api-version>
        <leveldb-version>0.12</leveldb-version>
        <leveldbjni-version>1.8</leveldbjni-version>
        <libphonenumber-version>8.13.3</libphonenumber-version>
        <libthrift-version>0.18.1</libthrift-version>
        <lifecycle-mapping-version>1.0.0</lifecycle-mapping-version>
        <lightcouch-version>0.2.0</lightcouch-version>
        <littleproxy-version>2.0.18</littleproxy-version>
        <log4j2-version>2.20.0</log4j2-version>
        <logback-version>1.4.8</logback-version>
        <lucene-version>9.7.0</lucene-version>
        <mapstruct-version>1.5.5.Final</mapstruct-version>
        <maven-antrun-plugin-version>3.1.0</maven-antrun-plugin-version>
        <maven-archetype-packaging-version>2.3</maven-archetype-packaging-version>
        <maven-archetype-plugin-version>3.2.1</maven-archetype-plugin-version>
        <maven-assembly-plugin-version>3.4.2</maven-assembly-plugin-version>
        <maven-jar-plugin-version>3.3.0</maven-jar-plugin-version>
        <maven-javadoc-plugin-version>3.4.1</maven-javadoc-plugin-version>
        <maven-owasp-plugin-version>8.3.1</maven-owasp-plugin-version>
        <maven-plugin-annotations-version>3.9.0</maven-plugin-annotations-version>
        <maven-plugin-plugin-version>3.9.0</maven-plugin-plugin-version>
        <maven-plugin-testing-harness-version>3.3.0</maven-plugin-testing-harness-version>
        <maven-reporting-api-version>2.2.1</maven-reporting-api-version>
        <maven-reporting-impl-version>2.0.5</maven-reporting-impl-version>
        <maven-resolver-version>1.9.13</maven-resolver-version>
        <maven-resources-plugin-version>3.3.1</maven-resources-plugin-version>
        <maven-shade-plugin-version>3.4.1</maven-shade-plugin-version>
        <maven-shared-utils-plugin-version>3.3.4</maven-shared-utils-plugin-version>
        <maven-surefire-report-plugin-version>3.0.0</maven-surefire-report-plugin-version>
        <maven-version>3.8.8</maven-version>
        <maven-wagon-version>3.5.2</maven-wagon-version>
        <maven-war-plugin-version>3.3.1</maven-war-plugin-version>
        <metrics-version>4.2.19</metrics-version>
        <micrometer-tracing-version>1.1.2</micrometer-tracing-version>
        <micrometer-version>1.11.1</micrometer-version>
        <microprofile-config-version>3.0.3</microprofile-config-version>
        <microprofile-fault-tolerance-version>4.0.2</microprofile-fault-tolerance-version>
        <mina-version>2.2.2</mina-version>
        <minidns-version>0.3.4</minidns-version>
        <minimal-json-version>0.9.5</minimal-json-version>
        <minio-version>8.5.3</minio-version>
        <mock-javamail-version>2.0</mock-javamail-version>
        <mockito-version>3.12.4</mockito-version>
        <mockwebserver-version>0.2.2</mockwebserver-version>
        <mongo-hadoop-version>1.5.0</mongo-hadoop-version>
        <mongo-java-driver-version>4.9.0</mongo-java-driver-version>
        <msal4j-version>1.13.8</msal4j-version>
        <mustache-java-version>0.9.10</mustache-java-version>
        <mvel-version>2.4.15.Final</mvel-version>
        <mybatis-version>3.5.13</mybatis-version>
        <mycila-license-version>4.2</mycila-license-version>
        <narayana-version>5.12.7.Final</narayana-version>
        <neoscada-version>0.4.0</neoscada-version>
        <nessus-hyperledger-aries-version>0.2.0</nessus-hyperledger-aries-version>
        <nessus-weka-version>1.0.1</nessus-weka-version>
        <netty-reactive-streams-version>2.0.5</netty-reactive-streams-version>
        <netty-version>4.1.94.Final</netty-version>
        <networknt-json-schema-validator-version>1.0.81</networknt-json-schema-validator-version>
        <nitrite-version>3.4.4</nitrite-version>
        <ognl-version>3.3.4</ognl-version>
        <okclient-version>3.14.9</okclient-version>
        <olingo2-version>2.0.11</olingo2-version>
        <olingo4-version>4.8.0</olingo4-version>
        <openjpa-version>3.2.2</openjpa-version>
        <opensearch-java-client-version>2.5.0</opensearch-java-client-version>
        <opensearch-rest-client-version>2.8.0</opensearch-rest-client-version>
        <opensearch-testcontainers-version>2.0.0</opensearch-testcontainers-version>
        <opensearch-version>2.8.0</opensearch-version>
        <openstack4j-version>3.11</openstack4j-version>
        <opentelemetry-alpha-version>${opentelemetry-version}-alpha</opentelemetry-alpha-version>
        <opentelemetry-version>1.26.0</opentelemetry-version>
        <optaplanner-version>9.37.0.Final</optaplanner-version>
        <os-maven-plugin-version>1.7.0</os-maven-plugin-version>
        <paho-version>1.2.5</paho-version>
        <parquet-avro-version>1.13.1</parquet-avro-version>
        <parquet-common-version>1.13.1</parquet-common-version>
        <pdfbox-version>2.0.28</pdfbox-version>
        <pgjdbc-driver-version>42.6.0</pgjdbc-driver-version>
        <pgjdbc-ng-driver-version>0.8.9</pgjdbc-ng-driver-version>
        <picocli-version>4.7.3</picocli-version>
        <plc4x-version>0.10.0</plc4x-version>
        <pooled-jms-version>3.1.0</pooled-jms-version>
        <powermock-version>2.0.7</powermock-version>
        <properties-maven-plugin-version>1.1.0</properties-maven-plugin-version>
        <proto-google-common-protos-version>2.21.0</proto-google-common-protos-version>
        <protobuf-maven-plugin-version>0.6.1</protobuf-maven-plugin-version>
        <protobuf-version>3.23.2</protobuf-version>
        <protonpack-version>1.8</protonpack-version>
        <protostream-version>4.6.2.Final</protostream-version>
        <prowide-version>SRU2022-10.0.1</prowide-version>
        <pubnub-version>6.3.5</pubnub-version>
        <pulsar-version>2.11.1</pulsar-version>
        <qpid-broker-version>9.0.0</qpid-broker-version>
        <qpid-jms-client-version>2.3.0</qpid-jms-client-version>
        <qpid-proton-j-version>0.34.1</qpid-proton-j-version>
        <quartz-version>2.3.2</quartz-version>
        <quickfixj-version>2.3.1</quickfixj-version>
        <reactive-streams-version>1.0.4</reactive-streams-version>
        <reactor-netty-version>1.1.6</reactor-netty-version>
        <reactor-version>3.5.6</reactor-version>
        <redisson-version>3.22.0</redisson-version>
        <resilience4j-version>2.0.2</resilience4j-version>
        <rest-assured-version>5.3.0</rest-assured-version>
        <roaster-version>2.28.0.Final</roaster-version>
        <robotframework-version>4.1.2</robotframework-version>
        <rocketmq-version>4.9.4</rocketmq-version>
        <rome-version>1.18.0</rome-version>
        <rxjava2-version>2.2.21</rxjava2-version>
        <saxon-version>12.2</saxon-version>
        <scala-datasonnet-version>2.13.10</scala-datasonnet-version>
        <servicemix-specs-version>2.9.0</servicemix-specs-version>
        <shiro-version>1.11.0</shiro-version>
        <slack-api-model-version>1.8.1</slack-api-model-version>
        <slf4j-api-version>2.0.6</slf4j-api-version>
        <slf4j-version>2.0.6</slf4j-version>
        <smack-version>4.3.5</smack-version>
        <smallrye-config-version>3.3.0</smallrye-config-version>
        <smallrye-fault-tolerance-version>6.2.4</smallrye-fault-tolerance-version>
        <smallrye-health-version>4.0.2</smallrye-health-version>
        <snakeyaml-engine-version>2.3</snakeyaml-engine-version>
        <snakeyaml-version>2.0</snakeyaml-version>
        <snmp4j-version>2.8.15</snmp4j-version>
        <splunk-version>1.9.0_1</splunk-version>
        <spock-version>2.3-groovy-4.0</spock-version>
        <spring-batch-version>5.0.2</spring-batch-version>
        <spring-data-redis-version>3.1.1</spring-data-redis-version>
        <spring-ldap-version>3.1.0</spring-ldap-version>
        <spring-rabbitmq-version>3.0.5</spring-rabbitmq-version>
        <spring-security-version>6.1.1</spring-security-version>
        <spring-vault-core-version>3.0.3</spring-vault-core-version>
        <spring-version>6.0.10</spring-version>
        <spring-ws-version>4.0.5</spring-ws-version>
        <sql-maven-plugin-version>1.5</sql-maven-plugin-version>
        <squareup-okhttp-version>3.14.9</squareup-okhttp-version>
        <squareup-okio-version>1.17.2</squareup-okio-version>
        <sshd-version>2.9.2</sshd-version>
        <stax-api-version>1.0.1</stax-api-version>
        <stompjms-version>1.19</stompjms-version>
        <stringtemplate-version>4.3.4</stringtemplate-version>
        <swagger-java-parser-version>1.0.65</swagger-java-parser-version>
        <swagger-java-version>1.6.10</swagger-java-version>
        <swagger-openapi3-java-parser-version>2.1.14</swagger-openapi3-java-parser-version>
        <swagger-openapi3-version>2.2.10</swagger-openapi3-version>
        <templating-maven-plugin-version>1.0.0</templating-maven-plugin-version>
        <testcontainers-version>1.17.6</testcontainers-version>
        <tika-version>2.8.0</tika-version>
        <twilio-version>9.6.2</twilio-version>
        <twitter4j-version>4.0.7</twitter4j-version>
        <undertow-version>2.3.7.Final</undertow-version>
        <univocity-parsers-version>2.9.1</univocity-parsers-version>
        <validation-api-version>2.0.1.Final</validation-api-version>
        <vavr-version>0.10.4</vavr-version>
        <velocity-tools-version>3.1</velocity-tools-version>
        <velocity-version>2.3</velocity-version>
        <vertx-version>4.4.4</vertx-version>
        <vysper-version>0.7</vysper-version>
        <web3j-quorum-version>0.8.0</web3j-quorum-version>
        <web3j-version>3.6.0</web3j-version>
        <wiremock-version>3.0.0-beta-9</wiremock-version>
        <woodstox-core-version>6.4.0</woodstox-core-version>
        <woodstox-version>4.4.1</woodstox-version>
        <xbean-asm5-shaded-version>4.5</xbean-asm5-shaded-version>
        <xbean-spring-version>4.21</xbean-spring-version>
        <xchange-version>5.0.14</xchange-version>
        <xerces-version>2.12.2</xerces-version>
        <xml-apis-ext-version>1.3.04</xml-apis-ext-version>
        <xml-apis-version>1.4.01</xml-apis-version>
        <xml-resolver-version>1.2</xml-resolver-version>
        <xmlgraphics-batik-version>1.16</xmlgraphics-batik-version>
        <xmlsec-version>2.2.3</xmlsec-version>
        <xmlunit-version>2.9.1</xmlunit-version>
        <xpp3-version>1.1.4c</xpp3-version>
        <yasson-version>3.0.3</yasson-version>
        <yetus-audience-annotations-version>0.13.0</yetus-audience-annotations-version>
        <zendesk-client-version>0.19.0</zendesk-client-version>
        <zookeeper-version>3.8.1</zookeeper-version>
        <zxing-version>3.5.0</zxing-version>
    </properties>

    <url>https://camel.apache.org</url>
    <inceptionYear>2007</inceptionYear>
    <scm>
        <connection>scm:git:http://gitbox.apache.org/repos/asf/camel.git</connection>
        <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/camel.git</developerConnection>
        <url>https://gitbox.apache.org/repos/asf?p=camel.git;a=summary</url>
        <tag>HEAD</tag>
    </scm>
    <issueManagement>
        <system>jira</system>
        <url>https://issues.apache.org/jira/browse/CAMEL</url>
    </issueManagement>

</project>
```

File: tooling/maven/sync-properties-maven-plugin/src/main/resources/camel-dependencies-template.xml (hunk 1 above)
Comment (ppalaga) [reply tooling/maven/sync-properties-maven-plugin/src/main/resources/camel-dependencies-template.xml:48]: I copied them from the top module without too much thinking. Given that I was able to cut a fake release, I assume these URLs really work.

**Pull request merged**